opencti-platform/opencti-graphql/tests/02-integration/02-resolvers/workspace-test.js

```diff
@@ -293,6 +293,30 @@ describe('Workspace resolver standard behavior', () => {
     });
   });
 
+  it('can duplicate workspace', async () => {
+    const queryResult = await queryAsAdmin({
+      query: gql`
+        mutation duplicateWorkspace($input: WorkspaceDuplicateInput!) {
+          workspaceDuplicate(input: $input){
+            id
+          }
+        }
+      `,
+      variables: {
+        input: {
+          type: 'dashboard',
+          name: 'Dashboard to duplicate',
+        },
+      },
+    });
+
+    expect(queryResult).not.toBeUndefined();
```

Comment:
can we test more data about the duplicated dashboard ? name, type, authorized members ?